### PR TITLE
adding WAREWULF_OCI vars to wwctl

### PR DESCRIPTION
**Pull request by mslacken**
Its only possible to supply the credetials for a registry with the
the environment variables `WAREWULF_OCI_USERNAME` and 
`WAREWULF_OCI_PASSWORD` which barely documented.
This PR add a command line parameters for `wwctl container import`

### CODE REVIEW

File: internal/pkg/api/container/container.go [hunk 1]
```diff
@@ -472,54 +472,42 @@ func ContainerRename(crp *wwapiv1.ContainerRenameParameter) (err error) {
...
+
+		}
+		// only set this if we want to disable, otherwise leave as undefined
+		if noHttps {
+			sCtx.DockerInsecureSkipTLSVerify = types.NewOptionalBool(true)
+		}
+		sCtx.OCIInsecureSkipTLSVerify = noHttps
+	}
+	if username != "" {
+		username, _ = os.LookupEnv("WAREWULF_OCI_USERNAME")
```
Comment (JasonYangShadow) [inline internal/pkg/api/container/container.go:496]: just to confirm, if the username is passed via CLI but the username is not set via `WAREWULF_OCI_USERNAME` (which means os.LookupEnv will return ''), the given username is still ignored here?

Comment (mslacken) [reply internal/pkg/api/container/container.go:496]: Right, it must be  `passwd == ""` so that we check for the env value if no cli value was given.

**Review by JasonYangShadow — APPROVED** [review]
lgtm!

**Review by anderbubble — CHANGES_REQUESTED** [review]
Please add documentation for the environment variables and the new arguments.

**Comment by mslacken** [comment]
@anderbubble Doucmentation is added now.

**Review by anderbubble — CHANGES_REQUESTED** [review]

**Comment by anderbubble** [comment]
Rebased to main.

**Comment by anderbubble** [comment]
I made a small change to use `--password` by default but retained `--passwd` as a hidden flag targeting the same variable. I tried using both (even together) and they behave reasonably (later value wins).

I tested this against CIQ's content delivery system "mountain" and it all worked exactly as expected. Thanks!

```
# wwctl container import --username ciq --password [redacted] docker://repository.ciq.com/warewulf-node-images/warewulf-node-images/fuzzball-substrate-compute:latest
Setting VNFS name: fuzzball-substrate-compute:latest
Copying blob a9567a1e64fc done   | 
Copying blob 13af37fe0be5 done   | 
Copying blob 0915b92be026 done   | 
Copying blob 88176b302c43 done   | 
Copying blob e1c015858fd3 done   | 
Copying blob 36a272932329 done   | 
Copying blob a5e464d3caa6 done   | 
Copying config 0a496c30a8 done   | 
Writing manifest to image destination
2024/09/04 15:55:03  info unpack layer: sha256:13af37fe0be55edef45f6a903677d6eff37426868dc8db800b153c647272d31b
2024/09/04 15:55:05  info unpack layer: sha256:88176b302c43bd8550b48f517d9cae140cbd5fe547aefebfe814d0da63d4d833
2024/09/04 15:55:10  info unpack layer: sha256:a9567a1e64fc5a5a55ba3dfe836cc0845df5abd77e153a7a5e0b9817dce92c55
2024/09/04 15:55:10  info unpack layer: sha256:0915b92be026154e144296d029d77a51754d56cebfa6cdf7eef20b5cb8a216fb
2024/09/04 15:55:10  info unpack layer: sha256:e1c015858fd30cc0090b39f4320caba6d36deab6dc55e466c19185a04b278336
2024/09/04 15:55:10  info unpack layer: sha256:36a272932329ab47eb454a22087a4b1212ea5bb52b6e669fae159b3e8322f083
2024/09/04 15:55:10  info unpack layer: sha256:a5e464d3caa69bc86e21247a5a29f1e990435d50c0414b03d5594debd3ebcab7
```

**Review by anderbubble — APPROVED** [review]

**Pull request merged**